src/psrose.c

```diff
@@ -160,59 +160,70 @@ static int usage (struct GMTAPI_CTRL *API, int level) {
-	GMT_Message (API, GMT_TIME_NONE, "\t-R Specifies the region (<r0> = 0, <r1> = max_radius).  For azimuth:\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Specify <theta0>/<theta1> = -90/90 or 0/180 (half-circles) or 0/360 only).\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   If <r0> = <r1> = 0, psrose will compute a reasonable <r1> value.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t-S Normalize data, i.e., divide all radii (or bin counts) by the maximum radius (or count).\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Optionally, append +a to normalize rose plots for area, i.e., take sqrt(r) before plotting [no area normalization].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t-T Indicate that the vectors are oriented (two-headed), not directed [Default].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   This implies both <azimuth> and <azimuth> + 180 will be counted as inputs.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Ignored if -R sets a half-circle domain.\n");
+	GMT_Usage (API, 1, "\n-Q<alpha>");
+	GMT_Usage (API, -2, "Set confidence level for Rayleigh test for uniformity [0.05].");
+	GMT_Usage (API, 1, "\n-R<r0>/<r1>/<theta0>/<theta1>");
+	GMT_Usage (API, -2, "Specify the region (<r0> = 0, <r1> = max_radius).  For azimuth: "
+		"Specify <theta0>/<theta1> = -90/90 or 0/180 (half-circles) or 0/360 only). "
+		"If <r0> = <r1> = 0, psrose will compute a reasonable <r1> value.");
+	GMT_Usage (API, 1, "\n-S[+a]");
+	GMT_Usage (API, -2, "Normalize data, i.e., divide all radii (or bin counts) by the maximum radius (or count). Optional modifier:");
+	GMT_Usage (API, 3, "+a Normalize rose plots for area, i.e., take sqrt(r) before plotting [no area normalization].");
+	GMT_Usage (API, 1, "\n-T Indicate that the vectors are oriented (two-headed), not directed [Default]. "
+		"This implies both <azimuth> and <azimuth> + 180 will be counted as inputs. "
+		"Ignored if -R sets a half-circle domain.");
 	GMT_Option (API, "U,V");
 	gmt_pen_syntax (API->GMT, 'W', NULL, "Set pen attributes for outline of rose [Default is no outline].", 0);
-	GMT_Message (API, GMT_TIME_NONE, "\t   Use -Wv<pen> to set a different pen for the vector (requires -E) [Same as rose outline].\n");
+	GMT_Usage (API, -2, "Note: Use -Wv<pen> to set a different pen for the vector (requires -E) [Same as rose outline].");
 	GMT_Option (API, "X");
-	GMT_Message (API, GMT_TIME_NONE, "\t-Z Multiply the radii by <scale> before plotting; use -Zu to set input radii to 1.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t-: Expect (azimuth,radius) input rather than (radius,azimuth) [%s].\n", choice[API->GMT->current.setting.io_lonlat_toggle[GMT_IN]]);
+	GMT_Usage (API, 1, "\n-Zu|<scale>");
+	GMT_Usage (API, -2, "Multiply the radii by <scale> before plotting; or use -Zu to set input radii to 1.");
+	GMT_Usage (API, -2, "\n-: Expect (azimuth,radius) input rather than (radius,azimuth) [%s].", choice[API->GMT->current.setting.io_lonlat_toggle[GMT_IN]]);
```

Comment:
```suggestion
	GMT_Usage (API, 1, "\n-: Expect (azimuth,radius) input rather than (radius,azimuth) [%s].", choice[API->GMT->current.setting.io_lonlat_toggle[GMT_IN]]);
```